src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs

```diff
@@ -133,6 +137,14 @@ public static ProjectFileInfo Create(MSBuildOptions options, ILogger logger, str
                 {
                     projectFileInfo.DefineConstants = defineConstants.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries).Distinct().ToList();
                 }
+
+                var signAssembly = projectInstance.GetPropertyValue("SignAssembly");
+                if (!string.IsNullOrWhiteSpace(signAssembly))
+                {
+                    projectFileInfo.SignAssembly = Convert.ToBoolean(signAssembly);
```

Comment:
I'm a bit concern about the exception here (https://msdn.microsoft.com/en-us/library/86hw82a3(v=vs.110).aspx). Do we want to fail everything in the case of an incorrect string representation or can we just let it slip and send a warning.

I'm fine with both, just make sure we're all aware of the possibility.